platform/admin/core/admin-core-insecuredefaults/src/test/java/org/codice/ddf/admin/insecure/defaults/service/KeystoreValidatorTest.java

```diff
@@ -161,8 +161,8 @@ public void testBlacklistKeystoreDoesNotExist() throws Exception {
                                 DEFAULT_KEYSTORE_PASSWORD),
                         String.format(KeystoreValidator.BLACKLIST_KEYSTORE_DOES_NOT_EXIST_MSG,
                                 path,
-                                FAKE_BLACK_LIST,
-                                FAKE_BLACK_LIST)};
+                                Paths.get(FAKE_BLACK_LIST),
```

Comment:
`FAKE_BLACK_LIST` was easily changed, but the other constants are tricky. They use resources that require multiple method calls and these methods throw exceptions I can't catch at variable declaration. Let's keep it the way it is with the single variable fixed.